app/controllers/application_controller/ci_processing.rb

```diff
@@ -1868,6 +1868,7 @@ def process_cloud_object_storage_buttons(pressed)
     task = pressed.sub("#{klass.name.underscore.to_sym}_", "")
 
     return tag(klass) if task == "tag"
+    return javascript_redirect(:controller => klass.name.underscore.to_sym, :action => "new") if task == "new"
 
```

Comment:
@miha-plesko All button handling for `cloud_object_store_container` should be in that controller itself, not in `ci_processing`
Check this out for instance --
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/cloud_network_controller.rb#L38-#L41